docs/source/08_integrations/1_prefect.md

```diff
@@ -0,0 +1,173 @@
+# Prefect
+
+Prefect is a workflow automation system. QHub integrates Prefect with a
+feature flag as follows:
+
+```yaml
+prefect: true
```

Comment:
at what level in the config yaml does this belong? went to another project to see that its at the zero indent level.

Reply:
At zero/top level. Anything that's not indented with another top level key is assumed to be top level.